src/controllers/expenses.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const expenseService = require('../services/expenses.js');
+const userService = require('../services/users.js');
+
+const getAll = (req, res) => {
```

Comment:
Don't ignore mentor comment, try to improve your code. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/70009205/3e3eedb4-67f0-4375-9cb3-11d5b0651663)